Dockerfile

```diff
@@ -20,7 +20,7 @@ FROM codait/max-base:v1.1.3
 RUN apt-get update && apt-get upgrade -y && rm -rf /var/lib/apt/lists/*
 
 # Fill in these with a link to the bucket containing the model and the model file name
-ARG model_bucket=https://max.cdn.appdomain.cloud/max-text-summarizer/1.0.0
+ARG model_bucket=https://max-cdn.appdomain.cloud/max-text-summarizer/1.0.0
```

Comment:
```suggestion
ARG model_bucket=https://max-cdn.cdn.appdomain.cloud/max-text-summarizer/1.0.0
```

Reply:
Hmm... both links work?

Reply:
OK, turns out that I have only examined the new link. It works perfectly.